.github/workflows/Tests.yml

```diff
@@ -72,16 +72,37 @@ jobs:
         with:
           version: '${{ matrix.version }}'
           arch: ${{ matrix.arch }}
-      - uses: actions/cache@v4
+      - uses: julia-actions/cache@v1
+      - uses: julia-actions/julia-buildpkg@v1
+      # TODO: Use julia-actions/julia-runtest when test_args are supported
+      # Custom calls of Pkg.test tend to miss features such as e.g. adjustments for CompatHelper PRs
+      # Ref https://github.com/julia-actions/julia-runtest/pull/73  
+      - name: Call Pkg.test
+        run: julia --color=yes --inline=yes --depwarn=yes --check-bounds=yes --threads=${{ matrix.num_threads }} --project=@. -e 'import Pkg; Pkg.test(; coverage=parse(Bool, ENV["COVERAGE"]), test_args=ARGS)' -- ${{ matrix.test-args }}
         env:
-          cache-name: cache-artifacts
+          COVERAGE: ${{ matrix.version == '1' && matrix.os == 'ubuntu-latest' && matrix.num_threads == 2 }}
```

Comment:
I tiny detail, but `${{ matrix.version == '1' && matrix.os == 'ubuntu-latest' && matrix.num_threads == 2 }}` could be stored in a variable and thus defined once, making sure the different if-clauses always use the same condition.